From a4a584e380aacad829bd37faa3df868c810ce7f1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADctor=20Manuel=20J=C3=A1quez=20Leal?=
 <vjaquez@igalia.com>
Date: Thu, 14 Sep 2023 20:17:06 +0200
Subject: [PATCH 198/232] msdk: meson: don't warn deprecated symbols

Since it's impossible to not to use any because any possible version mismatch
between the header and the runtime, such in the case of the denoise structure.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/5283>
---
 sys/msdk/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sys/msdk/meson.build b/sys/msdk/meson.build
index aaf3bdab7..51bf953a0 100644
--- a/sys/msdk/meson.build
+++ b/sys/msdk/meson.build
@@ -31,6 +31,7 @@ onevpl_extra_args = []
 
 extra_args = [
   '-DGST_USE_UNSTABLE_API',
+  '-DMFX_DEPRECATED_OFF'
 ]
 
 msdk_option = get_option('msdk')
-- 
2.40.1

